app/Main.hs

```diff
@@ -169,6 +178,7 @@ main = do
     ListPackages                          -> Spago.listPackages
     Sources                               -> Spago.sources
     Build limitJobs paths pursArgs        -> Spago.build limitJobs paths pursArgs
+    Verify limitJobs maybePackage         -> Spago.verify limitJobs maybePackage
```

Comment:
commented below, but i would really rather do away with the optional arguments and either add `verify-pkg packagename` or `verify-set`

Reply:
Current interface is:
```bash
# whole set
spago verify
# single package
spago verify simple-json
```

With `verify-set` it would be:
```bash
# whole set
spago verify-set
# single package
spago verify simple-json
```

I'm ok with having the `Maybe` there and skip adding a new command, as "less command surface" is better (less confusing for the user)